test/transactions/testSafeTransactions.ts

```diff
@@ -264,7 +286,7 @@ export function shouldSupportDifferentTransactions({
     });
 
     (
-      ownerIsRecognizedContract ? it.skip : it
+      !isCpkTransactionManager || ownerIsRecognizedContract ? it.skip : it
```

Comment:
Should be &&?

Reply:
Yes. However I removed `!isCpkTransactionManager` because this condition exists before calling this test file. Now when `!isCpkTransactionManager === true` the file testConnectedSafeTransactionsWithRelay is called.

Reply:
I don't think so